app/src/main/res/values-ja/strings.xml

```diff
@@ -30,4 +30,7 @@
         <item>print</item>
     </array>
 
+    <string name="please_wait">Please Wait</string>
+    <string name="populating_list">Populating list. This may take a while.</string>
```

Comment:
Some other user friendly term for this instead of 'Populating List'

Change it to.. 'Fetching files(or soemthing els)e. This may take a while'